lib/env_compare/cli.rb

```diff
@@ -103,5 +113,9 @@ def theme(file)
     def theme_path(file)
       @theme_path ||= File.join(File.dirname(__dir__), '..', 'themes', "#{file}.erb")
     end
+
+    def update_config(app, body)
```

Comment:
Maybe changing `body` to `hash`?

Reply:
Good idea; let me do that as a follow. I think I'm going to split these out into different files shortly

I'm going to use this in about 5 minutes 😂